src/Analyzers.sln

```diff
@@ -225,6 +225,14 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xUnit", "Dependencies\xUnit
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Security.Cryptography.Hashing.Algorithms.Analyzers.Setup", "System.Security.Cryptography.Hashing.Algorithms.Analyzers\Setup\System.Security.Cryptography.Hashing.Algorithms.Analyzers.Setup.csproj", "{989CDCD1-9031-4922-895B-8EB4D18D7A2D}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Text.Analyzers.Core (Net4)", "Text.Analyzers\Core (Net4)\Text.Analyzers.Core (Net4).csproj", "{703B2DE5-8131-4904-9CDD-BBD7708D9D36}"
```

Comment:
❗️ Remove spaces and special characters from the project paths

Reply:
No longer applies